eng/scripts/job-matrix/README.md

```diff
@@ -144,6 +147,24 @@ readable value here. For example:
 }
 ```
 
+#### $IMPORT
+
+Matrix configs can also import another matrix config. The effect of this is the imported matrix will be generated,
+and then the importing config will be combined with that matrix (as if each entry of the imported matrix was a parameter).
+To import a matrix, add a parameter with the key `$IMPORT`:
+
+```
+"matrix": {
+  "$IMPORT": "path/to/matrix.json",
+  "JavaVersion": [ "1.8", "1.11" ]
+}
+```
+
+Importing can be useful, for example, in cases where there is a shared base matrix, but there is a need to run it
+once for each instance of a language version.
+
+Includes and excludes for the importing matrix will still get processed after the imported matrix is combined with the base matrix.
```

Comment:
I think I'm fine with this behavior but I thought we were going to apply exclude/includes on the imported matrix before we cross it with the matrix that is importing it. Or is this yet another case where I'm confused by the terminology :) Let me go see the code...

Reply:
Yeah I need to work on wording. Here the "importing" matrix is different from the "imported" matrix 😁